packages/ui/src/components/editor/properties/model/index.tsx

```diff
@@ -108,7 +108,7 @@ export const ModelNodeEditor: EditorComponentType = (props) => {
       return
     }
     exporting.set(true)
-    const fileName = `${srcPath.value}.${exportType.value}`
+    const fileName = `assets/${srcPath.value}.${exportType.value}`
```

Comment:
we either need another argument to `exportRelativeGLTF` or to split the filename when passed into the `File` constructor as it shouldn't have the path in it

Reply:
Yes. My original solution was to modify exportRelativeGLTF to break this up.

This broke prefab export, as I had forgotten also references exportRelativeGLTF, where we are already including assets in the file, so David revertted it. Not wanting to make bigger core changes so close to the release, I just mirrored what we're already doing for prefabs and included assets in the relative path argument to make this not error out.